examples/reference-manifest/IpStb/firebolt-device-manifest.json

```diff
@@ -0,0 +1,265 @@
+      ]
+    }
+  },
+  "capabilities": {
+    "supported": [
+      "xrn:firebolt:capability:accessory:pair",
+      "xrn:firebolt:capability:accessory:list",
+      "xrn:firebolt:capability:accessibility:closedcaptions",
+      "xrn:firebolt:capability:accessibility:voiceguidance",
+      "xrn:firebolt:capability:account:id",
+      "xrn:firebolt:capability:account:uid",
+      "xrn:firebolt:capability:accessory:list",
+      "xrn:firebolt:capability:advertising:configuration",
+      "xrn:firebolt:capability:advertising:identifier",
+      "xrn:firebolt:capability:approve:content",
+      "xrn:firebolt:capability:approve:purchase",
+      "xrn:firebolt:capability:data:app-usage",
+      "xrn:firebolt:capability:device:distributor",
+      "xrn:firebolt:capability:device:id",
+      "xrn:firebolt:capability:device:info",
+      "xrn:firebolt:capability:device:make",
+      "xrn:firebolt:capability:device:model",
+      "xrn:firebolt:capability:device:name",
+      "xrn:firebolt:capability:device:sku",
+      "xrn:firebolt:capability:device:uid",
+      "xrn:firebolt:capability:discovery:entity-info",
+      "xrn:firebolt:capability:discovery:navigate-to",
+      "xrn:firebolt:capability:discovery:policy",
+      "xrn:firebolt:capability:discovery:purchased-content",
+      "xrn:firebolt:capability:discovery:watched",
+      "xrn:firebolt:capability:discovery:watch-next",
+      "xrn:firebolt:capability:discovery:content-access",
+      "xrn:firebolt:capability:grants:state",
+      "xrn:firebolt:capability:lifecycle:initialize",
+      "xrn:firebolt:capability:lifecycle:launch",
+      "xrn:firebolt:capability:lifecycle:ready",
+      "xrn:firebolt:capability:lifecycle:state",
+      "xrn:firebolt:capability:localization:country-code",
+      "xrn:firebolt:capability:localization:language",
+      "xrn:firebolt:capability:localization:locale",
+      "xrn:firebolt:capability:localization:locality",
+      "xrn:firebolt:capability:localization:postal-code",
+      "xrn:firebolt:capability:localization:time-zone",
+      "xrn:firebolt:capability:metrics:general",
+      "xrn:firebolt:capability:metrics:media",
+      "xrn:firebolt:capability:network:status",
+      "xrn:firebolt:capability:power:state",
+      "xrn:firebolt:capability:privacy:advertising",
+      "xrn:firebolt:capability:privacy:content",
+      "xrn:firebolt:capability:privacy:settings",
+      "xrn:firebolt:capability:profile:flags",
+      "xrn:firebolt:capability:usergrant:pinchallenge",
+      "xrn:firebolt:capability:usergrant:acknowledgechallenge",
+      "xrn:firebolt:capability:input:keyboard",
+      "xrn:firebolt:capability:remote:ble",
+      "xrn:firebolt:capability:storage:secure"
+    ],
+    "grantPolicies": {
+      "xrn:firebolt:capability:account:id": {
+        "use": {
+          "options": [
+            {
+              "steps": [
+                {
+                  "capability": "xrn:firebolt:capability:usergrant:fingerprint"
```

Comment:
dnt think we have support for finger print

Reply:
removed mentioned feature.